### ISSUE TITLE
drop externalAttributeFilter.js now that we have experimental babel support

### ISSUE THREAD

**Issue by cdaringe** [Tripwire/octagon#193]
# problem

`externalAttributeFilter.js` is useful, but no longer needed once 11.x kicks out

we can now do this instead

```jsx
function Comp (props) {
  const {
     thing,
     thang,
     children,
     ...rest
  }
  return (
     <p {...rest}>
        {thing ? x(thang) : null}
        {children}
     </p>
  )
}
```

using the object rest spread is more complete, safer (more explicit prop handling), and faster. 💯

**Comment by youngbob** [comment]
Looks like this was addressed in 4a4d47a

**Comment by cdaringe** [comment]
agreed.
